config/sidekiq.yml

```diff
@@ -67,3 +67,7 @@
       interval: 1 hour
       class: Scheduler::AutoCloseRegistrationsScheduler
       queue: scheduler
+    ip_spamlist_url_scheduler:
```

Comment:
Do we need to contemplate the scenario where a person wants to NOT BLOCK an IP from these lists?